docker-compose.dev.yml

```diff
@@ -8,6 +8,7 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
       - AUTH0_AUDIENCE=${AUTH0_AUDIENCE}
       - AUTH0_ISSUER=${AUTH0_ISSUER}
+      - MAIL_PASSWORD=${MAIL_PASSWORD}
```

Comment:
could you elaborate on why you added this environment variable? and what it does.

Reply:
This is the mail password for accessing the gmail server to send emails.